src/main/java/com/google/cloud/dataproc/templates/pubsub/README.md

```diff
@@ -35,3 +35,52 @@ pubsub.bq.output.table=<bq output table>
 ## Number of records to be written per message to BigQuery
 pubsub.bq.batch.size=1000
 ```
+
+## 2. Pub/Sub To GCS
+
```

Comment:
We need to provide details about assumption and requirements for pubsub message payload format etc if any.
It seems like we are requiring customer to have JSON as payload but that is nowhere mentioned in readme.
Also how those column names are mapped to GCS / Avro columns and what happens if there is array / map / collections in the json payload.

Reply:
Hi @shashank-google , this solution supports AVRO and JSOn data formats. It does not require customer to follow any particular schema.  If AVRO data is passed through pubsub topic then it will be stored as .avro files in GCS.If the  pubsub topic had JSON data passed through it, we can mention JSON as the file format on solution and it will produce .json files in GCS. Hope this clarifies.